docs/concepts/jcloud/yaml-spec.md

```diff
@@ -149,11 +149,19 @@ executors:
 
 ### Storage
 
-JCloud supports two kinds of storage types: [efs](https://aws.amazon.com/efs/) (default) and [ebs](https://aws.amazon.com/ebs/). The former is a network file storage, whereas the latter is a block device.
+JCloud supports three kinds of storage types: ephemeral (default), [efs](https://aws.amazon.com/efs/) and [ebs](https://aws.amazon.com/ebs/). `efs` is a network file storage, whereas the `ebs` is a block device.
+
+`ephemeral` storage will assign space to Executor when it created, and exists as long as that Pod is running on that node. When an Executor is removed or rescheduled for any reason, the data in the `ephemeral` storage is deleted permanently.
```

Comment:
Terms like `Pod`, `Node` etc might be irrelevant to Jina users. We can say,

It is a temporary storage that grows & shrinks automatically, but data might be lost if Executors are restarted.